modules/@apostrophecms/schema/ui/apos/components/AposInputRelationship.vue

```diff
@@ -317,6 +320,18 @@ export default {
       if (this.field.editor === 'AposImageRelationshipEditor') {
         return 'apostrophe:editImageAdjustments';
       }
+    },
+    getDefault() {
+      const object = {};
+      this.field.schema.forEach(field => {
+        // Do not simply check if `field.def` is truthy, this in not enough
+        // since a def as an empty string must be considered valid:
+        const hasDefaultValue = Object.prototype.hasOwnProperty.call(field, 'def');
```

Comment:
Use `Object.hasOwn` (not `hasOwnProperty`). It's supported across the board in modern browsers and the syntax is much simpler. Server side we'll get access to it in two weeks when Node 14 dies 😄